### PR TITLE
[API-1517]: Add option to create Web SDK Viewer HTML

### PR DESCRIPTION
**Pull request by therockstorm**

### CODE REVIEW

File: src/commands/render-image.ts [hunk 1]
```diff
@@ -81,3 +104,63 @@ async function createFile(
...
+      nomodule
+      src="https://unpkg.com/@vertexvis/viewer@0.9.x/dist/viewer.js"
+    ></script>
+  </head>
+  <body>
+    <vertex-viewer id="viewer" class="viewer" style="width: auto; height: auto">
+    </vertex-viewer>
+
+    <script type="module">
+      import { ColorMaterial } from 'https://unpkg.com/@vertexvis/viewer@0.9.x/dist/viewer/index.esm.js';
```
Comment (danschultz) [inline src/commands/render-image.ts:130]: Will follow up, but this isn’t the recommended url anymore.

Comment (danschultz) [reply src/commands/render-image.ts:130]: Here's the URL: https://unpkg.com/@vertexvis/viewer@0.9.x/dist/esm/index.mjs

File: src/commands/render-image.ts [hunk 2]
```diff
@@ -81,3 +104,63 @@ async function createFile(
...
+      type="module"
+      src="https://unpkg.com/@vertexvis/viewer@0.9.x/dist/viewer/viewer.esm.js"
+    ></script>
+    <script
+      nomodule
+      src="https://unpkg.com/@vertexvis/viewer@0.9.x/dist/viewer.js"
+    ></script>
+  </head>
+  <body>
+    <vertex-viewer id="viewer" class="viewer" style="width: auto; height: auto">
```
Comment (danschultz) [inline src/commands/render-image.ts:126]: We'll be requiring a client ID in the near future. I think we should add that here as well. Note, recommendation would be to have a different client ID than what is used by internal APIs.

Comment (therockstorm) [reply src/commands/render-image.ts:126]: So this is the actual applications client id? What is the key? `clientId="xxx"`?

Comment (danschultz) [reply src/commands/render-image.ts:126]: > So this is the actual applications client id?

Yep.

> What is the key?

`<vertex-viewer client-id="123">`

**Review by danschultz — APPROVED** [review]

**Pull request merged**